### ISSUE TITLE
Radio buttons have no focus styles

### ISSUE THREAD

**Issue by smcmurtry** [veteransaffairscanada/vac-benefits-directory#1343]
This is an issue in /guided and /benefits-directory. It's is possible to operate this feature using only the keyboard, but the lack of focus style makes this very difficult.

**Comment by smcmurtry** [comment]
Closed by #1361
